src/rdfTypes/GeometryInfoHelpersImpl.h

```diff
@@ -38,6 +48,15 @@ inline std::string removeDatatype(const std::string_view& wkt) {
   return std::string{asStringViewUnsafe(lit.getContent())};
 }
 
+// Adds quotation marks and the `geo:wktLiteral` datatype to a given string
+inline std::string addDatatype(const std::string_view wkt) {
+  auto lit = ad_utility::triple_component::Literal::literalWithoutQuotes(wkt);
+  auto dt = ad_utility::triple_component::Iri::fromIrirefWithoutBrackets(
+      GEO_WKT_LITERAL);
+  lit.addDatatype(dt);
+  return lit.toStringRepresentation();
```

Comment:
At least `std::move` out the string representation (this is rather inefficient if used in a tight loop).